.github/CODEOWNERS

```
@@ -37,6 +37,7 @@ dd-smoke-tests/debugger-integration-tests/ @DataDog/debugger-java
 dd-java-agent/agent-iast/              @DataDog/asm-java
 dd-java-agent/instrumentation/*iast*   @DataDog/asm-java
 dd-java-agent/instrumentation/*appsec* @DataDog/asm-java
+dd-java-agent/instrumentation/json/    @DataDog/asm-java
```

Can you also include the following?

```suggestion
dd-java-agent/instrumentation/json/    @DataDog/asm-java
dd-smoke-tests/iast-util/              @DataDog/asm-java
dd-java-agent/instrumentation/commons-fileupload/    @DataDog/asm-java
```